example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java

```diff
@@ -32,9 +34,23 @@ public static void main(String[] args) throws ClassNotFoundException, SQLExcepti
     try (Connection connection = DriverManager.getConnection("jdbc:iotdb://127.0.0.1:6667/", "root", "root");
          Statement statement = connection.createStatement()) {
       statement.execute("SET STORAGE GROUP TO root.sg1");
-      statement.execute("CREATE TIMESERIES root.sg1.d1.s1 WITH DATATYPE=INT64, ENCODING=RLE");
-      statement.execute("CREATE TIMESERIES root.sg1.d1.s2 WITH DATATYPE=INT64, ENCODING=RLE");
-      statement.execute("CREATE TIMESERIES root.sg1.d1.s3 WITH DATATYPE=INT64, ENCODING=RLE");
+      try {
+        statement.execute("CREATE TIMESERIES root.sg1.d1.s1 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
+      } catch (IoTDBSQLException e) {
+        System.out.println(e.getMessage());
+      }
+
+      try {
+        statement.execute("CREATE TIMESERIES root.sg1.d1.s3 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
+      } catch (IoTDBSQLException e) {
+        System.out.println(e.getMessage());
+      }
+
+      try {
+        statement.execute("CREATE TIMESERIES root.sg1.d1.s2 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
+      } catch (IoTDBSQLException e) {
+        System.out.println(e.getMessage());
+      }
```

Comment:
Hi, I think one try-catch is enough. Catch the set storage group and create timeseries.

Reply:
Hi，The original one is better，I'm sorry I did some useless jobs

Reply:
Hi~ Actually, I mean add a try-catch for set storage group and create timeseries